examples/wallet-connect.html

```diff
@@ -139,10 +139,15 @@ <h1>Success</h1>
         method: "Query",
         protocol: "http://profile-protocol.xyz",
       },
+      {
+        interface: "Records",
+        method: "Read",
+        protocol: "http://profile-protocol.xyz",
+      },
     ];
 
     try {
-      const { delegateDid } = await Web5.connect({
+      const { delegateDid, delegateGrants } = await Web5.connect({
```

Comment:
delegateGrants won't exist as discussed in the connect comment. Also the references below.

```suggestion
      const { delegateDid } = await Web5.connect({
```